app/src/main/res/drawable/cfr_shape_with_top_arrow.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Use `android:autoMirrored="true"` to have this automatically flipped in RTL and so prevent
![image](https://user-images.githubusercontent.com/11428869/152801510-6332864c-b4d1-496f-9183-abf8ba3a0a7f.png)